lib/multipass_ex/crypto.ex

```diff
@@ -0,0 +1,127 @@
+defmodule MultipassEx.Crypto do
+  @moduledoc """
+  Module for encrypting and decrypting data in compliance with the Ruby MultiPass
+  gem.
+
+  AES_CBC128 is the encryption algorithm used.
+
+  __Warning__: The ruby gem uses an insecure IV and it's mirrored here to be compatible
+  since it is not sent with the Multipass data.
```

Comment:
Would it be worth allowing the user to specify their own initialization vector even though it's not rails compatible?